open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-spring/src/main/java/org/odpi/openmetadata/governanceservers/openlineage/server/spring/OpenLineageResource.java

```diff
@@ -32,15 +32,15 @@
      * @param guid         The guid of the node of which the lineage is queried of.
      * @return A subgraph containing all relevant paths, in graphSON format.
      */
-    @GetMapping(path = "/initial-graph/{scope}/{lineageQuery}/{graph}/{guid}")
-    public String initialGraph(
+    @GetMapping(path = "/query-lineage/{scope}/{lineageQuery}/{graph}/{guid}")
+    public String queryLineage(
             @PathVariable("userId") String userId,
             @PathVariable("serverName") String serverName,
             @PathVariable("scope") String scope,
             @PathVariable("lineageQuery") String lineageQuery,
             @PathVariable("graph") String graph,
             @PathVariable("guid") String guid) {
-        return restAPI.initialGraph(serverName, userId, scope, lineageQuery, graph, guid);
+        return restAPI.queryLineage(serverName, userId, scope, lineageQuery, graph, guid);
     }
 
     /**
```

Comment:
Noticed class inTopicListener starts with lower i, please rename it to start with upper i

Reply:
Fixed, thanks for spotting